bynder_sdk/util.py

```diff
@@ -17,23 +19,29 @@ def parse_json_for_response(response):
 
 
 class SessionMixin:
+    # pylint:disable=keyword-arg-before-vararg
     def wrapped_request(self, func, endpoint, *args, **kwargs):
+        need_response_json = kwargs.pop("need_response_json", True)
         endpoint = api_endpoint_url(self, endpoint)
         response = func(endpoint, *args, **kwargs)
         response.raise_for_status()
-
+        if not need_response_json:
+            return response
```

Comment:
Instead of introducing this `need_response_json` arg, couldn't we just check somehow in the `response` object if it has a json body?

Or maybe something like this would do the trick?
```
try:
    return response.json()
except:
    return response
```
(and ditch parse_json_for_response() since we wouldn't need it anymore)

Reply:
Good idea 😄